builtin/logical/pki/cert_util.go

```diff
@@ -91,28 +91,30 @@ func getFormat(data *framework.FieldData) string {
 
 // Fetches the CA info. Unlike other certificates, the CA info is stored
 // in the backend as a CertBundle, because we are storing its private key
-func fetchCAInfo(ctx context.Context, b *backend, req *logical.Request) (*certutil.CAInfoBundle, error) {
-	bundleEntry, err := req.Storage.Get(ctx, "config/ca_bundle")
+func fetchCAInfo(ctx context.Context, b *backend, req *logical.Request, issuerRef string) (*certutil.CAInfoBundle, error) {
+	id, err := resolveIssuerReference(ctx, req.Storage, issuerRef)
```

Comment:
I'm not clear that this structure can work as with discussion of today about functioning, that this structure of get reference + below fetchCertBundleByIssuerId would have to be different to enable "get the old one"

Reply:
Yeah, I think migration concerns need to be addressed more concisely in one place :/